va/meson.build

```diff
@@ -70,8 +70,8 @@ libva = shared_library(
   version : libva_lt_version,
   c_args : [ '-DSYSCONFDIR="' + sysconfdir + '"'] + ['-DVA_DRIVERS_PATH="' + driverdir + '"'] + va_c_args,
   include_directories : configinc,
-  link_args : '-Wl,-version-script,' + libva_sym_path,
-  link_depends : libva_sym,
+  link_args : host_machine.system() == 'windows' ? [] : '-Wl,-version-script,' + libva_sym_path,
```

Comment:
what's the operation priority here? do we risk to end up with just `+ libva_sym_path`?

Reply:
I checked on Ubuntu/MSVC and the build command line was correct. Just to make sure a future meson change doesn't break this, I added the parentheses around it to make it explicit